### PR TITLE
Feature/692 editor update

### PR DESCRIPTION
**Pull request by JrgenGNielsen**

### CODE REVIEW

File: ding_content.make [hunk 1]
```diff
@@ -48,8 +48,12 @@ projects[views][version] = "3.8"
 projects[workbench][subdir] = "contrib"
 projects[workbench][version] = "1.2"
 
+projects[wysiwyg][type] = "module"
 projects[wysiwyg][subdir] = "contrib"
-projects[wysiwyg][version] = "2.2"
+projects[wysiwyg][download][type] = "git"
+projects[wysiwyg][download][url] = "http://git.drupal.org/project/wysiwyg.git"
+projects[wysiwyg][download][revision] = "7981731f4f3db2f932419499d2ec13a073e9b88f"
+projects[wysiwyg][download][branch] = "7.x-2.x"
```
Comment (kasperg) [inline ding_content.make]: When we provide a revision then branch is not used/needed. Please remove it.

File: ding_content.make [hunk 2]
```diff
@@ -48,8 +48,12 @@ projects[views][version] = "3.8"
 projects[workbench][subdir] = "contrib"
 projects[workbench][version] = "1.2"
 
+projects[wysiwyg][type] = "module"
 projects[wysiwyg][subdir] = "contrib"
-projects[wysiwyg][version] = "2.2"
+projects[wysiwyg][download][type] = "git"
+projects[wysiwyg][download][url] = "http://git.drupal.org/project/wysiwyg.git"
+projects[wysiwyg][download][revision] = "7981731f4f3db2f932419499d2ec13a073e9b88f"
```
Comment (kasperg) [inline ding_content.make]: Please add a comment stating why this revision is used. This is useful for determining if we can switch to a stable version later on.

**Comment by kasperg** [comment]
@JrgenGNielsen Review complete.

I assume that you have tested that the editor and WYSIWYG profile still work as expected.

File: ding_content.make (hunk 2 above)
Comment (JrgenGNielsen) [inline ding_content.make]: As I read it, the [module owner recommend using the dev version] (https://www.drupal.org/node/1853550#comment-9213657)

**Comment by JrgenGNielsen** [comment]
As far as I can tell, the changes look merely cosmetic. But it'll need a test by someone who's used to using the editor

File: ding_content.make (hunk 2 above)
Comment (kasperg) [inline ding_content.make]: OK. Then please add a comment explaining why a specific revision is used.

**Comment by kasperg** [comment]
Reviewed and approved. Thanks!

**Pull request merged**